extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -109,8 +109,7 @@ public function getTopContributorsData() {
 	 * @return array
 	 */
 	public function getTopAdminsData() {
-		$allAdmins = $this->getAllAdmins();
-		$allAdminsDetails = $this->getContributorsDetails( $allAdmins );
+		$allAdminsDetails = $this->getContributorsDetails( $this->usersModel->getAllAdmins() );
```

Comment:
I'm sceptical as we'll end up caching additional data like username and avatar which will increase number of places where we should purge this. Fortunately user and avatar are already cached, so I think performance gain is not worth the complexity increase here.

Although I'm planning to reduce number of users passed to getContributorsDetails as we don't need to walk through the whole list here.